code/modules/reagents/chemistry/recipes/medicine.dm

```diff
@@ -264,3 +264,27 @@
 	id = /datum/reagent/medicine/psicodine
 	results = list(/datum/reagent/medicine/psicodine = 5)
 	required_reagents = list( /datum/reagent/medicine/mannitol = 2, /datum/reagent/water = 2, /datum/reagent/impedrezene = 1)
+
+/datum/chemical_reaction/burnmix // Chemistry recipe
+	name = "BurnMix"
```

Comment:
I don't understand why this is being changed. Can you inform me before I commit?
Looking at other medicine in the same file they are structured with the name itself being a string rather than a reference.

Reply:
this isn't how names work alex it'd show up as /datum/reagent/medicine/burnmix in game

and decapitalize this, by the way